Add procedures to migrate from RHEL8to9 using backups (#3103)
theforeman/foreman-documentation: guides/common/modules/con_upgrading-el-on-project-or-proxy.adoc

Reviewer: Isn't Leapp a third method? Do you plan to update this concept with info about #3070?

[id="upgrading_EL_on_{project-context}_or_proxy_{context}"]
= Upgrading {EL} on {Project} or {SmartProxy}

> There are two methods to upgrade your {Project} or {SmartProxy} from {EL} 8 to {EL} 9:
> * Using LEAPP
> * Backup and restore

Reviewer: ```suggestion
There are two methods to upgrade your {Project} or {SmartProxy} from {EL} 8 to {EL} 9:

* Leapp in-place upgrade
* Backup and restore
```

> With Leapp you can upgrade your {Project} or {SmartProxy} in place and hence it is faster, however this imposes a downtime on the services.

Reviewer: ```suggestion
With Leapp, you can upgrade your {Project} or {SmartProxy} in place and, therefore, it is faster but imposes a downtime on the services.
```

> Using backup allows you to move your {Project} or {SmartProxy} to a fresh {EL} 9 system.

Reviewer: ```suggestion
With backup, you can move your {Project} or {SmartProxy} to a fresh {EL} 9 system.
```

[id="upgrading_EL_on_{project-context}_or_proxy_{context}"]
= Upgrading {EL} on {Project} or {SmartProxy}

There are two methods to upgrade your {Project} or {SmartProxy} from {EL} 8 to {EL} 9:

* Leapp in-place upgrade

> * Backup and restore

Reviewer: ```suggestion
* Migration by using backup and restore
```

[id="upgrading_EL_on_{project-context}_or_proxy_{context}"]
= Upgrading {EL} on {Project} or {SmartProxy}

{Project} and {SmartProxy} are supported on both {EL} 8 and {EL} 9.
You can upgrade your {Project} and {SmartProxy} operating system from {EL} 8 to {EL} 9.

There are two methods to upgrade your {Project} or {SmartProxy} from {EL} 8 to {EL} 9:

* Leapp in-place upgrade
* Migration by using backup and restore

With Leapp, you can upgrade your {Project} or {SmartProxy} in place therefore it is faster but imposes a downtime on the services.

> With backup, you can move your {Project} or {SmartProxy} to a fresh {EL} 9 system.
> Backup requires very little downtime.

Reviewer 2: ```suggestion
With migration, you can move your {Project} or {SmartProxy} to a fresh {EL} 9 system.
The {EL} 8 system remains operational during the migration, which reduces the downtime.
```
